src/Sulu/Bundle/AdminBundle/Resources/js/stores/ResourceStore/ResourceStore.js

```diff
@@ -39,12 +45,23 @@ export default class ResourceStore {
     }
 
     @action setLocale(locale: string) {
-        this.locale.set(locale);
+        const {locale: observableLocale} = this.observableOptions;
+        if (!observableLocale) {
+            // TODO Should there really be a silent return, or should we throw an exception instead?
```

Comment:
@chirimoya What do you think about that? What should happen if `setLocale` on the `ResourceStore` is called, if there is no locale to set? Should we maybe change the interface completely?